index.php

```diff
@@ -53,26 +58,28 @@
         </div>
 
         <div class="row p-3">
+            <?php foreach ($tasks as $task) : ?>
             <div class="col-sm-6 col-md-4 col-lg-3 py-3 py-3">
                 <div class="card">
                     <img src="https://picsum.photos/200" class="card-img-top" alt="...">
                     <div class="card-body">
-                        <h5 class="card-title">ここにタイトル</h5>
+                        <h5 class="card-title"><?= h($task["title"]); ?></h5>
                         <p class="card-text">
-                            ここに詳細
+                            <?= h($task["contents"]); ?>
                         </p>
                         <div class="text-right d-flex justify-content-end">
                             <!-- * href内を変更する -->
-                            <a href="edit.php" class="btn text-success">EDIT</a>
+                            <a href="edit.php?id=<? h($task['id']); ?>" class="btn text-success">EDIT</a>
```

Comment:
=がぬけているのでecho できていないですね！
（なのでURLにidが入っていなかったはず）

```suggestion
                            <a href="edit.php?id=<?= h($task['id']); ?>" class="btn text-success">EDIT</a>
```